org.civicrm.bootstrapcivicrm/js/civihr-popup.js

```diff
@@ -0,0 +1,62 @@
+/*
+Prevent Popups to overflow tables.
```

Comment:
How does this logic deals with multiple popups? What if i click on a popup and then immediately click on another one without first closing the other?